### PR TITLE
add jax.linalg.numpy.norm to jax frontend

### LINKED ISSUE #5758: norm

### PR DESCRIPTION
**Pull request by mohame54**
close #5758

### CODE REVIEW

**Comment by mohame54** [comment]
I don't know what the problem is I have tested it many times  on my local machine and everything was ok

**Comment by kimbring2** [comment]
@mohame54 Hello Mohame54. Thank you for your contribution. 

Your code looks fine except for the max_axes_size. It should be 1 to pass the test. The current value sometimes gives the (0, 3) kind of axis value which gives an error!

**Comment by mohame54** [comment]
thanks a a lot

**Comment by mohame54** [comment]
but can I ask you a question? how did you recognize this detail ?

**Comment by kimbring2** [comment]
@mohame54 Oh, that is because I also wrote the norm function for the Numpy frontend. lol

**Comment by mohame54** [comment]
Oh I see, thank god I have you here

**Comment by mohame54** [comment]
Oh I see, thank god I have you here

**Comment by mohame54** [comment]
but there is a problem

**Comment by mohame54** [comment]
I have deleted this forked repo and started a new one again

**Comment by kimbring2** [comment]
@mohame54 Yes, ok
